Sources/Build/describe().swift

```diff
@@ -103,6 +103,65 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
         }
     }
 
+    //For C language Modules
+    //FIXME: Probably needs more compiler options for debug and release modes
+    //FIXME: Incremental builds
+    //FIXME: Add support for executables
+    for case let module as ClangModule in modules {
+        
+        //FIXME: Generate modulemaps if possible
+        //Since we're not generating modulemaps currently we'll just emit empty module map file
+        //if it not present
+        if !module.moduleMapPath.isFile {
+            try mkdir(module.moduleMapPath.parentDirectory)
+            try fopen(module.moduleMapPath, mode: .Write) { fp in
+                try fputs("\n", fp)
+            }
+        }
+        
+        let inputs = module.dependencies.map{ $0.targetName } + module.sources.paths
+        let productPath = Path.join(prefix, "lib\(module.c99name).so")
+        let wd = Path.join(prefix, "\(module.c99name).build")
+        mkdirs.insert(wd)
+        
+        var args: [String] = []
+    #if os(Linux)
+         args += ["-fPIC"]
```

Comment:
That is probably correct behavior, I would want to test it with Swift interop to know for sure.

> On Mar 10, 2016, at 11:04 AM, Ankit Agarwal <notifications@github.com> wrote:
> 
> In Sources/Build/describe().swift <https://github.com/apple/swift-package-manager/pull/183#discussion_r55731599>:
> 
> > +        //if it not present
> > +        if !module.moduleMapPath.isFile {
> > +            try mkdir(module.moduleMapPath.parentDirectory)
> > +            try fopen(module.moduleMapPath, mode: .Write) { fp in
> > +                try fputs("\n", fp)
> > +            }
> > +        }
> > +        
> > +        let inputs = module.dependencies.map{ $0.targetName } + module.sources.paths
> > +        let productPath = Path.join(prefix, "lib\(module.c99name).so")
> > +        let wd = Path.join(prefix, "\(module.c99name).build")
> > +        mkdirs.insert(wd)
> > +        
> > +        var args: [String] = []
> > +    #if os(Linux)
> > +         args += ["-fPIC"]
> Tests were failing on linux and clang suggested to recompile using this. Not sure if this is correct? @mxcl <https://github.com/mxcl> @ddunbar <https://github.com/ddunbar>
> —
> Reply to this email directly or view it on GitHub <https://github.com/apple/swift-package-manager/pull/183/files#r55731599>.
>